src/basic/terminal-util.h

```diff
@@ -124,18 +144,36 @@ bool dev_console_colors_enabled(void);
 
 DEFINE_ANSI_FUNC(normal,            NORMAL);
 DEFINE_ANSI_FUNC(highlight,         HIGHLIGHT);
+DEFINE_ANSI_FUNC(black,             BLACK);
 DEFINE_ANSI_FUNC(red,               RED);
 DEFINE_ANSI_FUNC(green,             GREEN);
 DEFINE_ANSI_FUNC(yellow,            YELLOW);
 DEFINE_ANSI_FUNC(blue,              BLUE);
 DEFINE_ANSI_FUNC(magenta,           MAGENTA);
+DEFINE_ANSI_FUNC(cyan,              CYAN);
+DEFINE_ANSI_FUNC(white,             WHITE);
 DEFINE_ANSI_FUNC(grey,              GREY);
+
+DEFINE_ANSI_FUNC(bright_black,      BRIGHT_BLACK);
+DEFINE_ANSI_FUNC(bright_red,        BRIGHT_RED);
+DEFINE_ANSI_FUNC(bright_green,      BRIGHT_GREEN);
+DEFINE_ANSI_FUNC(bright_yellow,     BRIGHT_YELLOW);
+DEFINE_ANSI_FUNC(bright_blue,       BRIGHT_BLUE);
+DEFINE_ANSI_FUNC(bright_magenta,    BRIGHT_MAGENTA);
+DEFINE_ANSI_FUNC(bright_cyan,       BRIGHT_CYAN);
+DEFINE_ANSI_FUNC(bright_white,      BRIGHT_WHITE);
+
+DEFINE_ANSI_FUNC(highlight_black,   HIGHLIGHT_BLACK);
 DEFINE_ANSI_FUNC(highlight_red,     HIGHLIGHT_RED);
 DEFINE_ANSI_FUNC(highlight_green,   HIGHLIGHT_GREEN);
-DEFINE_ANSI_FUNC(highlight_yellow,  HIGHLIGHT_YELLOW);
+DEFINE_ANSI_FUNC(highlight_yellow_original,
+                                    HIGHLIGHT_YELLOW);
+DEFINE_ANSI_FUNC(highlight_yellow,  HIGHLIGHT_YELLOW_REPLACEMENT);
```

Comment:
it's a bit weird that the constant uses yellow/yellow-replacement and the wrapper function uses yellow-original/yellow as names.

I think either we should always use "yellow-original" or "yellow-replacement"; or we should use "yellow-original" and "yellow" (and only mention the "replacement" in a comment next to it).